packages/editor-ui/src/plugins/i18n/locales/en.json

```
@@ -1026,6 +1027,11 @@
 	"nodeErrorView.theErrorCauseIsTooLargeToBeDisplayed": "The error cause is too large to be displayed",
 	"nodeErrorView.time": "Time",
 	"nodeErrorView.inputPanel.previousNodeError.title": "Error running node '{nodeName}'",
+	"nodeErrorView.debugError.button": "Ask AI ✨",
+	"nodeErrorView.debugError.loading": "Asking AI.. ✨",
+	"nodeErrorView.debugError.feedback.positive": "I found this helpful",
+	"nodeErrorView.debugError.feedback.negative": "I didn't find this helpful",
```

As we have already the general versions below of this we can just reference it:
```suggestion
	"nodeErrorView.debugError.feedback.positive": "@:feedback.positive",
	"nodeErrorView.debugError.feedback.negative": "@:feedback.negative",
```

---

Removed these since they're no longer used. Good catch!